src/locale/locales/fr/messages.po

```diff
@@ -7828,7 +7564,7 @@ msgstr "Vous ne suivez personne."
 
 #: src/components/dialogs/nuxs/NeueTypography.tsx:61
 msgid "You can adjust these in your Appearance Settings later."
-msgstr ""
+msgstr "Vous pourrez les ajuster plus tard dans les paramètres d’apparence."
```

Comment:
```suggestion
msgstr "Vous pourrez les ajuster plus tard dans les paramètres d’affichage."
```
We translated `appearance settings` previously as `paramètres d’affichage` (L691 + L695). Should we stick with that or alter the other strings to `paramètres d’apparence`?